pkg/kindsys/report.go

```diff
@@ -53,47 +55,123 @@ var plannedCoreKinds = []string{
 	"QueryHistory",
 }
 
+type Kind struct {
```

Comment:
Probably we can switch back again to `Properties`, or embed them into report's `Kind`. 

Although, I'm not sure, if all these properties (different name variants, etc) are meaningful for the sake of the report. If so, we may want to diff between meta and "dimensions".